src/frontend/src/optimizer/plan_node/stream_sink.rs

```diff
@@ -150,13 +149,6 @@ impl StreamSink {
             Distribution::Single => RequiredDist::single(),
             _ => {
                 match properties.get("connector") {
-                    Some(s) if s == "deltalake" => {
-                        // iceberg with multiple parallelism will fail easily with concurrent commit
-                        // on metadata
-                        // TODO: reset iceberg sink to have multiple parallelism
```

Comment:
What's the parallelism requirement  for iceberg rust version

Reply:
The comment was for previous iceberg sink. Now there is no singleton requirement on both iceberg sink and delta lake sink.